internal/providers/registry.go

```diff
@@ -56,6 +61,10 @@ type ProviderInfo struct {
 	LastIndex cid.Cid
 	// LastIndexTime is the time indexed data was added to the indexer
 	LastIndexTime time.Time
+
+	// lastContactTime is the time of last contact or attempted contact with
```

Comment:
Remove?

Reply:
I am going to need it for polling, but it may end up being a timer... so will remove for now.